artifacts/definitions/Windows/EventLogs/PowershellModule.yaml

```diff
@@ -35,21 +35,14 @@ parameters:
     description: "regex search over Payload text field."
   - name: SearchVSS
     description: "Add VSS into query."
-    type: bool
-  - name: LogLevelMap
```

Comment:
Why is this removed? It is needed below.

Reply:
@scudette For the PowershellModule content I took the PowershellScriptblock and copy pasta'd.
Module Logging doesnt have that log level setting so I removed it from the query but noticed I left it in the parameter definitions. I missed the VQL which was what you noticed here.  I just added a commit to update and remove the VQL.